tsk/fs/logical_fs.cpp

```diff
@@ -85,6 +85,90 @@ filetime_to_timet(FILETIME const& ft)
 }
 #endif
 
+/**
+* Check if the given path contains the folder separator
+* 
+* @param path  The path to test
+* 
+* @return true if path contains folder separator, false otherwise
+*/
+static bool
+contains_folder_separator(const TSK_TCHAR* path) {
+	if (path == NULL) {
+		return false;
+	}
+
+#ifdef TSK_WIN32
+	TSK_TCHAR slash = '\\';
```

Comment:
/ is also a valid path separator in Windows in many contexts and its an invalid filename character so I think its safe to look for both on windows